wgengine/packet/header.go

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) 2020 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package packet
+
+import "errors"
+
+// Declared here for the lack of a TCPHeader struct.
```

Comment:
If you're going to leave a godoc on something, use the normal format and start by saying what it is, not just why it's here.

I'd also be fine with just removing this comment. It's unexported so doesn't need docs if it's obvious. And no need to justify placement of internal things really.